src/test/java/com/sports/server/auth/aop/AuthenticationAspectTest.java

```diff
@@ -0,0 +1,126 @@
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.web.context.request.RequestContextHolder;
+import org.springframework.web.context.request.ServletRequestAttributes;
+
+@SpringBootTest
+@DatabaseIsolation
+@Import(TestSecurityConfig.class)
+@Sql(scripts = "/member-fixture.sql")
+public class AuthenticationAspectTest {
+
+    @Value("${cookie.name}")
+    private String COOKIE_NAME;
+
+    @Value("${jwt.secret-key}")
+    private String secretKey;
+
+    @Value("${jwt.valid-time}")
+    private long TOKEN_VALID_TIME;
+
+    @Autowired
+    private MemberRepository memberRepository;
+
+    @Autowired
+    private JwtUtil jwtUtil;
+
+    @Mock
+    private ProceedingJoinPoint proceedingJoinPoint;
+
+    @Mock
+    private MethodSignature methodSignature;
+
+    private AuthenticationAspect authenticationAspect;
+    private MockHttpServletRequest request;
+    private Member member;
+    private String email;
+    private Cookie cookie;
+    private String accessToken;
+
+    @BeforeEach
+    void setUp() {
+        MockitoAnnotations.openMocks(this);
+        request = new MockHttpServletRequest();
+        email = "john@example.com";
+        member = memberRepository.findMemberByEmail(email).get();
+        accessToken = jwtUtil.createAccessToken(member);
+        cookie = new Cookie(COOKIE_NAME, accessToken);
+        request.setCookies(cookie);
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(request)); // RequestContextHolder에 설정
+        authenticationAspect = new AuthenticationAspect(memberRepository, jwtUtil);
+
+        when(proceedingJoinPoint.getSignature()).thenReturn(methodSignature);
+        when(methodSignature.getMethod()).thenReturn(this.getClass().getDeclaredMethods()[0]);
+    }
+
+    @Test
+    void 검증에_성공한다() throws Throwable {
+        when(proceedingJoinPoint.proceed(any())).thenReturn(member);
+        Member resultMember = (Member) authenticationAspect.authenticate(proceedingJoinPoint, null);
+
+        assertNotNull(resultMember);
+        assertEquals(email, resultMember.getEmail());
+    }
+
+    @Test
+    void 존재하지_않는_회원일_경우_예외를_던진다() {
+        String fakeToken = JWT.create()
+                .withSubject("fake@gmail.com")
+                .withExpiresAt(new Date(new Date().getTime() + TOKEN_VALID_TIME))
+                .withClaim("id", 1L)
+                .withClaim("email", "fake@gmail.com")
+                .sign(Algorithm.HMAC512(secretKey));
+
+        Cookie fakeCookie = new Cookie(COOKIE_NAME, fakeToken);
+        request.setCookies(fakeCookie);
+        RequestContextHolder.setRequestAttributes(new ServletRequestAttributes(request));
+
+        UnauthorizedException exception = assertThrows(UnauthorizedException.class, () -> {
+            authenticationAspect.authenticate(proceedingJoinPoint, null);
+        });
+        assertEquals(AuthorizationErrorMessages.MEMBER_NOT_FOUND_EXCEPTION, exception.getMessage());
```

Comment:
``assertThatThrownBy()`` 메서드로 예외 테스트를 한줄로 줄일 수 있을 거 같아.

```suggestion
    assertThatThrownBy(
        () -> authenticationAspect.authenticate(proceedingJoinPoint, null)
    ).isInstanceOf(UnauthorizedException.class)
    .hasMesssage(AuthorizationErrorMessages.MEMBER_NOT_FOUND_EXCEPTION);
```